bevm/bevm_call_byzcoin_test.go

```diff
@@ -257,8 +258,7 @@ func Test_BEvmCallsByzcoin(t *testing.T) {
 	valID := byzcoin.ComputeNewInstanceID(myValueContractID, seed)
 
 	// Check that the new instance exists and holds the correct value
-	_, err = cl.WaitProof(valID, 10*time.Second, initValue)
-	require.NoError(t, err)
+	require.True(t, instanceIDExists(t, cl, valID, initValue))
```

Comment:
OK - I didn't think you want to check the value. So you could do this and wouldn't need an external method:

```
_, err = cl.WaitProof(valID, 0, initValue)
require.NoError(t, err)
```